src/components/v2/values/chartValuesDiff/ChartValuesView.utils.ts

```diff
@@ -181,7 +181,11 @@ const getAvailalbePath = (parentPathKey: string[], valuesYamlDocument: YAML.Docu
         } else {
             const _currentPath = currentPath.concat(_pathKey)
             const _valueInCurrentPath = valuesYamlDocument.getIn(_currentPath)
-            if (typeof _valueInCurrentPath === 'undefined' || _valueInCurrentPath === null) {
+            if (
+                typeof _valueInCurrentPath === 'undefined' ||
+                _valueInCurrentPath === null ||
+                (_valueInCurrentPath.items && !_valueInCurrentPath.items.length)
```

Comment:
can we write _valueInCurrentPath.items && !_valueInCurrentPath.items.length to !_valueInCurrentPath.items?.length?

Reply:
No, because it'll give undefined and go in if condition for normal objects. We just want to go in if condition when a value has the `items` key and it doesn't have any element. For normal objects, we have different handling.